pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -99,6 +102,7 @@ Listen Sockets
   * ``tcpListenQueueSize=SOMAXCONN``: int - Set the size of the listen queue. Default is ``SOMAXCONN``.
   * ``maxInFlight=0``: int - Maximum number of in-flight queries. The default is 0, which disables out-of-order processing.
   * ``maxConcurrentTCPConnections=0``: int - Maximum number of concurrent incoming TCP connections. The default is 0 which means unlimited.
+  * ``allowProxyProtocol=true``: str - Whether to allow a proxy protocol v2 header in front of incoming queries. Both this option and :func:`setProxyProtocolACL` needs to agree for a payload to be accepted. Default is ``true``, meaning that queries are allowed if they come from an address present in the :func:`setProxyProtocolACL` ACL.
```

Comment:
is it really "allow"? does it still accept packets without a proxy header?

Reply:
It's a very good point! No, it doesn't accept packets without a proxy protocol header, so the name is wrong.

Reply:
I just checked the auth for inspiration, but it uses `proxy-protocol-from`, which is not a form that helps you. Maybe something as simple as `enableProxyProtocol`?

Reply:
`enableProxyProtocol` sounds good, thanks.

Reply:
Done!